src/ast-parse/astParse.ts

```diff
@@ -67,74 +83,141 @@ export async function astParseRoot (
   const parsingResults: ParsingResult = {}
   const transformationResults: AstTransformationResult = {}
 
-  const transformationParams: TransformationParams = {
-    config: config
-  }
-  cliInstance.setTotal(cliInstance.total + resolvedPaths.length)
```

Comment:
We should not delete this line, the total should be plus the number of ast parsing times the number of files.